docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -204,6 +211,15 @@ We can send an http request directly to the Broker with correct CloudEvent heade
 
       exit
       ```
+
+**Note** : For v0.18 or above, use `http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default` as the address instead of `default-broker.knative-samples.svc.cluster.local`. You can view the address by:
```

Comment:
Instead of adding a note here, we should just replace `default-broker.knative-samples.svc.cluster.local` with `http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default`throughout the docs site, instead of mentioning specific versions in the text. Users should refer to the correct version of docs for their installation. This change should be applied to `master` for 0.19 and also cherrypicked to 0.18.

Reply:
@gwind the oldest version of docs on the site now is 0.18 so this note could be removed and the address should just be updated to `http://broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default` throughout.

Please lmk if you don't intend to complete this PR so that we can reassign the work if it's still needed.

cc @pierDipi for eventing